doc/src/modules/algebras.rst

```diff
@@ -0,0 +1,19 @@
+===============
+Algebras Module
+===============
+
+Introduction
+------------
+
+The algebras module for SymPy will provide support
```

Comment:
The algebras module for SymPy **provides** support
for basic **algebraic** operations on Quaternions.